shared/haml/hamburger.haml

```diff
@@ -92,17 +110,24 @@
 
 #hamburger{class: hamburger_class}
   %ul.hide-responsive-menu
-    .user-links
+    .user-links{class: show_user_options}
```

Comment:
`show_user_options` seems redundant since it just wraps some more specific types that we show/hide.